### ISSUE TITLE
[Bug]: Factorio unable to download files

### ISSUE THREAD

**Issue by RichardAnderson** [parkervcp/eggs#1588]
### Panel Version

1.7

### Wings Version

1.6.1

### Service

factorio

### Modified

No, I did not modify the egg

### Expected Behavior

Factorio installs successfully

### Actual Behavior

Factorio does not install - the files are not downloaded
https://ibb.co/hcLYm0T

Changing the CURL request to the below:

`curl --show-error --location --silent https://www.factorio.com/get-download/${DL_VERSION}/headless/linux64 -o factorio-${DL_VERSION}`

Outputs the following:
`running 'curl --show-error --location --silent https://www.factorio.com/get-download/1.1.53/headless/linux64 -o factorio-1.1.53' 
curl: (6) Could not resolve host: dl.factorio.com`

This works fine on the host, but not on the docker child.  Docker child is able to evaluate factorio.com but not dl.factorio.com.  Unable to replicate on any other guest docker image.

`root@docker-test:~# ls
check-config.sh  gpinstall  snap
root@docker-test:~# curl --show-error --location --silent https://www.factorio.com/get-download/1.1.53/headless/linux64 -o factorio-1.1.53
root@docker-test:~# ls
check-config.sh  factorio-1.1.53  gpinstall  snap
root@docker-test:~#`

### Steps To Reproduce

Step 1 - Install a new Server with Factorio Egg
Step 2 - Navigate to management page of Server

### Install logs

http://bin.ptdl.co/awq2n

**Comment by Software-Noob** [comment]
This isn't an egg-related issue as your container can't resolve domains. It's most likely the Fastly DNS bug (magical). Try restarting Docker.

**Comment by RichardAnderson** [comment]
Hi - thanks for this, I've restarted the service and server multiple times, but its still not working.
I think i've gotten to the bottom of it, and it might be a Pterodactyl issue.

If I run:
# docker run -ti ghcr.io/parkervcp/installers:debian bash
# cat /etc/resolv.conf
search openstacklocal
nameserver 172.17.0.1
nameserver 8.8.4.4
nameserver 8.8.8.8
#

However, if I create an egg with this, I get the following:
RESOLV.CONF
search openstacklocal 
nameserver 127.0.0.11 
options edns0 ndots:0

So basically, I get two different outputs from running the docker container through shell, and another when running the container through Pterodactyl - would there be any reason for this?

**Comment by RichardAnderson** [comment]
Issue resolved.  Problem with Docker configuration and dnsmasq on Ubuntu 18.04 (and 20).
Steps to resolve as below:

Shut down all running containers

```
# mkdir /etc/NetworkManager/dnsmasq.d
# echo "listen-address=172.18.0.1" >> /etc/NetworkManager/dnsmasq.d/pterodactyl-bridge.conf
# sudo systemctl stop wings
# nano /etc/pterodactyl/config.yml
```

Update the DNS resolvers - the above listener is needed for network manager, otherwise it will block the docker request for these nameservers, and the default will be used. 

REPLACE:
```
docker:
  network:
    interface: 172.18.0.1
    dns:
    - 1.1.1.1
    - 1.1.1.2
    name: pterodactyl_nw
```

WITH:
```
docker:
  network:
    interface: 172.18.0.1
    dns:
    - 8.8.8.8
    - 8.8.4.4
    name: pterodactyl_nw
```

Restart the server - or restart network services, docker and wings

**Comment by jaschuu** [comment]
why can we not use 1.1.1.1 and 1.1.1.2?

**Comment by RichardAnderson** [comment]
No idea - for some reason these are not working on default installs of Ubuntu 18 and 20 both at Digital Ocean and Intovps - had to change them to google DNS.

**Comment by jaschuu** [comment]
I am using Unraid, I changed my router and unraids network settings to 8.8.8.8 and 8.8.4.4 and it seems to be working now, I dislike having to do this.
